src/recordlinker/hl7/fhir.py

```diff
@@ -27,13 +27,21 @@ def fhir_record_to_pii_record(fhir_record: dict) -> schemas.PIIRecord:
         "race": None,
         "gender": None,
         "telecom": fhir_record.get("telecom", []),
+        "drivers_license": None,
     }
     for identifier in fhir_record.get("identifier", []):
         for coding in identifier.get("type", {}).get("coding", []):
             if coding.get("code") == "MR":
                 val["mrn"] = identifier.get("value")
             elif coding.get("code") == "SS":
                 val["ssn"] = identifier.get("value")
+            elif coding.get("code") == "DL":
+                license_number = identifier.get("value")
+                authority = identifier.get("assigner", {}).get("identifier", {}).get("value")  # Assuming `issuer` contains authority info
```

Comment:
should we add an if block here to only add a DL if both license_number and authority are non-null?

Reply:
oh I see, down below we only yield the value if they are defined, so I guess we don't need that if block